### PR TITLE
feat: uuid-ossp extensions

### PR DESCRIPTION
**Pull request by yechoi42**
> What was wrong/why is this fix needed? (quick summary only)

주요 변경사항
- postgresql 에 uuid-ossp extension 설치(init_db.sh) : 설치에 관한 자세한 설명은 아래 이슈 참고
- 환경변수 .env로 분리

> Issue number (if there is an associated issue)
#78
> Anything else that could help us review it

### CODE REVIEW

**Review by yhshin0 — APPROVED** [review]

File: back/ormconfig.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
   "username": "user",
   "password": "password",
   "database": "database",
-  "entities": ["dist/**/**.entity{.ts,.js}"],
+  "entities": [
+    "dist/**/**.entity{.ts,.js}"
+  ],
   "synchronize": true
-}
+}
```
Comment (hochan222) [inline back/ormconfig.json:12]:
```suggestion
}

```
eol

File: db/.env [hunk 2]
```diff
@@ -0,0 +1,4 @@
+POSTGRES_USER=user
+POSTGRES_PORT=5432
+POSTGRES_PASSWORD=password
+POSTGRES_DB=database
```
Comment (hochan222) [inline db/.env:4]:
```suggestion
POSTGRES_DB=database

```

File: db/init_db.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+    GRANT ALL PRIVILEGES ON DATABASE "$POSTGRES_DB" TO "$POSTGRES_USER";
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+EOSQL
```
Comment (hochan222) [inline db/init_db.sh:8]:
```suggestion
EOSQL

```

**Review by hochan222 — APPROVED** [review]
LGTM :)

**Review by jwon42 — APPROVED** [review]

**Pull request merged**